CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ A [BC BREAK] means the update will break the project for many reasons:
 * new dependencies
 * class refactoring
 
+### 2015-10-05
+
+* Update [IvoryCKEditorBundle](https://github.com/egeloen/IvoryCKEditorBundle) to version 3.x. Please consider to replace ```{{ ckeditor_replace() }}``` twig function to ```{{ ckeditor_widget() }}```
```

Comment:
Add `[BC BREAK]` prefix and remove the second sentence (please consider...), this part should be on the upgrade notice.